### PR TITLE
feat: Add nodeSelector and tolerantions option to promitor-agent-scraper

### PR DESCRIPTION
**Pull request by blafry**
Hello,

i've added these options to allow pod assignments to specific nodes.

### CODE REVIEW

**Review by tomkerkhove — COMMENTED** [review]
Can you change this in both charts please?

Thanks a ton for your contribution!

PS: Feel free to revert the version Rev for now

**Comment by blafry** [comment]
Sure, I will add to second one too.

> PS: Feel free to revert the version Rev for now

I'm not sure what do you mean, change the revision to `Rev`?

**Comment by tomkerkhove** [comment]
I was referring to the chart version, sorry for not being clear!

**Comment by blafry** [comment]
@tomkerkhove added also to promitor-agent-resource-discovery chart and version reverted.

**Review by tomkerkhove — COMMENTED** [review]
One more request, sorry for forgetting.

Could you update the Readme of the charts please? Sorry 😕

**Comment by blafry** [comment]
> Could you update the Readme of the charts please?

Sure, done :)

File: promitor-agent-resource-discovery/templates/deployment.yaml [hunk 1]
```diff
@@ -28,6 +28,13 @@ spec:
       {{- if .Values.rbac.create }}
       serviceAccountName: {{ template "promitor-agent-resource-discovery.serviceaccountname" . }}
       {{- end }}
+      nodeSelector:
```
Comment (tomkerkhove) [inline promitor-agent-resource-discovery/templates/deployment.yaml:31]: We are missing an if here:
```suggestion
      {{- if .Values.nodeSelector }}
      nodeSelector:
```

**Review by tomkerkhove — APPROVED** [review]
LGTM, just need to fix the linting issue & sign the CLA - Thanks a ton for this contribution!

**Comment by tomkerkhove** [comment]
Thank you for the contribution @blafry!

**Pull request merged**